addons/ingress-nginx/addon.rb

```diff
@@ -13,6 +13,7 @@
       'image' => 'registry.pharos.sh/kontenapharos/pharos-default-backend:0.0.3'
     )
     attribute :tolerations, Pharos::Types::Array.default([])
+    attribute :enable_dynamic_certificates, Pharos::Types::Bool.default(false)
```

Comment:
`enable_dynamic_certificates` vs `dynamic_certificates` 🤔

Reply:
Tried to "model" it 1-to-1 with the upstream options

Reply:
What if we just allow to set `extra_args` (array of arguments)? It would be much more generic...

Reply:
Sure.